#Envelope to send a priority serialized ROS message through the mesh system
Header header
string host
string type
uint8 priority
string topic
uint8[] data